dist/components/tabs/_tabs.dust

```diff
@@ -0,0 +1,20 @@
+{?tabs}
+    <div class="c-tabs" role="tab">
+        <ul id="c-tab-controls" class="c-tabs__controls" role="tablist">
+            {#tabs}
+                <li class="c-tabs__controls-item {class}">
+                    <button class="c-tabs__button" data-target="{id}" aria-label="{title}">{title}</button>
```

Comment:
Nope, defaults to type="submit", which if it’s in a form (hello ASP.net!) it will trigger the form submission without `event.preventDefault();`. So adding the button type is always a good idea.